src/services/user/services/AdminUsers.js

```diff
@@ -87,7 +88,10 @@ class AdminUsers {
 
 			// fetch data that are scoped to schoolId
 			const searchedRole = roles.find((role) => role.name === this.role);
-			return getAllUsers(schoolId, currentYear, searchedRole._id, query);
+
+			const users = await getAllUsers(schoolId, currentYear, searchedRole._id, query);
+			this.formatBirthdayOfUsers(users.data);
```

Comment:
This is the perfect use case for an after-hook

Reply:
done